cluster_test.go

```diff
@@ -1693,20 +1693,47 @@ func (s *S) TestMaxSocketUses(c *C) {
 		time.Sleep(5e8)
 	}
 
-	// a socket was used during connection
-	numUses := 1
+	for i := 0; i < 2; i++ {
+		stats = mgo.GetStats()
+		// every 4th usage should result in a new connection
```

Comment:
I don't plan on removing maxSocketUses. This was result of copy/paste test case where I accidentally modified original test case.

Now that I think about it I think probably it's safe to remove as I didn't find any references of maxSocketUse in the code base. I think it's not used anywhere. I'll let Bill comment on it.